docker/images/ubuntu_common/build.sh

```diff
@@ -42,13 +42,15 @@ mkdir -p $DESTINATION
 
 echo "bootstrapping a build environment..."
 debootstrap --variant=buildd  \
+            # following line fixes deboostrap for buster, possible to delete if it works without it
+            --include="perl"  \
             --arch=$BASE_ARCH \
 	          --force-check-gpg \
             $UBUNTU_RELEASE   \
             $DESTINATION      \
             $REPO_BASE_URL
 
-if [ x"$UBUNTU_RELEASE" != x"bionic" ]; then
+if [ x"$UBUNTU_RELEASE" != x"bionic" ] && [ x"$UBUNTU_RELEASE" != x"buster" ] ; then
```

Comment:
Is this step also needed for buster?  I.e., did it not work without the modification?

Reply:
Yes, one of the repositories below in the code is not available and the job crashes: https://epsft-jenkins.cern.ch/job/CvmfsFullBuildDocker/CVMFS_BUILD_ARCH=docker-x86_64,CVMFS_BUILD_PLATFORM=debian10/1037/console